assets/javascripts/discourse-user-notes/templates/connectors/admin-user-controls-after/add-user-notes-button.hbs

```diff
@@ -1 +1 @@
-{{show-user-notes show=(action "showUserNotes") count=userNotesCount}}
```

Comment:
Looks like there are a couple more connector templates which could be modernised. In particular I'm thinking of the `{{action` modifiers here:

https://github.com/discourse/discourse-user-notes/blob/1fc0b0241f5ecfc94814bd40b02eb316e507e2ac/assets/javascripts/discourse-user-notes/templates/connectors/user-card-post-names/show-user-notes-on-card.hbs#L2

https://github.com/discourse/discourse-user-notes/blob/1fc0b0241f5ecfc94814bd40b02eb316e507e2ac/assets/javascripts/discourse-user-notes/templates/connectors/after-reviewable-post-user/show-user-notes-on-flags.hbs#L2